docs/source/user_guide/computed_properties.rst

```diff
@@ -95,12 +95,15 @@ In the right panel of SimStack, go to ``Jobs & Workflows``. Completed jobs (show
 
 .. _viz:
 
-Morphology Files & Visualization
------------------------------
+Generated Morphology
```

Comment:
Obviously the morphology is Generated, otherwise we could not work with it. I'd prefer the old title, also bc it contains the word "visualization" which then appears in the menu which seems to be important for Franz